libr/debug/p/native/procfs.c

```diff
@@ -0,0 +1,31 @@
+/* radare - LGPL - Copyright 2009-2016 - pancake */
+
+#include <r_userconf.h>
+#include <r_debug.h>
+
+int procfs_pid_slurp(int pid, char *prop, char *out, size_t len) {
+	char *filename;
+	int fd, ret = -1;
+	ssize_t nr;
+
+	filename = r_str_newf ("/proc/%d/%s", pid, prop);
```

Comment:
r_file_slurp allocates dynamic memory. the current method here uses stack buffers. it's a minor difference really, but eases memory management overhead.